index.html

```diff
@@ -370,11 +370,12 @@ <h2>
       <pre class="idl">
         [Constructor(WakeLockType type), SecureContext, Exposed=(DedicatedWorker, Window)]
         interface WakeLock : EventTarget {
-          [Exposed=Window] static Promise&lt;PermissionState&gt; requestPermission(WakeLockType type);
           readonly attribute WakeLockType type;
           readonly attribute boolean active;
           attribute EventHandler onactivechange;
           Promise&lt;void> request(optional WakeLockRequestOptions options);
+          static FrozenArray&lt;WakeLock> query(optional WakeLockQueryFilter filter);
```

Comment:
sequence&lt;WakeLock>

Reply:
As far as I understand from reading Web IDL, sequence does a copy and FrozenArray allows to store references instead @littledan

Reply:
Maybe it is fine that new objects are created as they are backed by the same underlying platform lock, so will have same state etc.

The only issue that I see is that the events handlers will not be reflected

Reply:
I don't think FrozenArray will implicitly cache and return the same Array; see https://heycam.github.io/webidl/#es-frozen-array . `sequence<WakeLock>` SGTM.

Reply:
Yes, I see that that algorithm just creates a sequence and then freezes it, but I got confused by this

![image](https://user-images.githubusercontent.com/1101339/55480373-71d16000-5620-11e9-9d11-f6369ac68500.png)

which talks about references vs values passes by values

Reply:
Yeah, I agree that this text is confusing; cc @ms2ger